Sources/Common/DataModel/Box/index.js

```diff
@@ -82,12 +83,45 @@ function intersectBox(bounds, origin, dir, coord, tolerance) {
   return 1;
 }
 
+// Plane intersection with box
+// The plane is infinite in extent and defined by an origin and normal.The function indicates
+// whether the plane intersects, not the particulars of intersection points and such
+// The function returns non-zero if the plane and box intersect; zero otherwise.
+function intersectPlane(bounds, origin, normal) {
+  const p = [];
+  let d = 0;
+  let sign = 1;
+  let firstOne = 1;
```

Comment:
here you could use `sign=undefined`  to know if it is the first time is has been set